README.md

```diff
@@ -92,47 +92,97 @@ The milestone needs testing on all platforms. Please report issues when found.
-cmake --preset linux -D ENABLE_LOGGER=[ON|OFF]
-cmake --build --preset linux --config [Debug|Release]
-```
-
-#### Building - Linux (Fedora)
-```sh
-sudo dnf update --refresh
-sudo dnf install -y ninja-build cmake gcc-c++ SDL2-devel zlib-devel
-cmake --preset linux -D ENABLE_LOGGER=[ON|OFF]
-cmake --build --preset linux --config [Debug|Release]
-```
+1. Make sure that any prerequisites are installed:
+
+    <details>
+    <summary><b>Windows</b></summary>
+
+    You’ll need Git and Visual Studio 2022 with the “Desktop development with C++” workload. If you don’t already have those installed or you aren’t sure, then open an elevated Command Prompt and run:
+
+    <!--
+    The following code block specifies the full path to the Visual Studio Installer because the Visual Studio Installer doesn’t add itself to the user’s Path. The installer is guaranteed to be in a specific location on 64-bit systems [1]. The installer will be in a different location on 32-bit systems [2], but Visual Studio 2022 doesn’t support 32-bit systems [3] so we can ignore that detail.
+
+    [1]: <https://learn.microsoft.com/en-us/visualstudio/install/use-command-line-parameters-to-install-visual-studio?view=vs-2022>
+    [2]: <https://github.com/microsoft/vswhere/wiki#installing>
+    [3]: <https://learn.microsoft.com/en-us/answers/questions/1689898/does-visual-studio-build-tools-2022-support-32-bit>
+    -->
+
+    ```batch
+    winget install Git.Git Microsoft.VisualStudio.2022.Community
+
+    "%ProgramFiles(x86)%\Microsoft Visual Studio\Installer\setup.exe" modify^
+        --passive^
+        --channelId VisualStudio.17.Release^
+        --productId Microsoft.VisualStudio.Product.Community^
+        --add Microsoft.VisualStudio.Workload.NativeDesktop;includeRecommended
+    ```
+
+    </details>
+
+    <details>
+    <summary><b>macOS</b></summary>
+
+    1. Make sure that [Xcode](https://developer.apple.com/xcode) is installed.
+
+    2. Make sure that [Homebrew](https://brew.sh) is installed.
+
+    </details>
+
+    <details>
+    <summary><b>Ubuntu</b></summary>
+
+    Make sure that the required packages are installed:
+
+    ```sh
+    sudo apt update
+    sudo apt install -y --no-install-recommends git ninja-build cmake g++ libsdl2-dev zlib1g-dev
+    ```
+
+    </details>
+
+    <details>
+    <summary><b>Fedora</b></summary>
+
+    Make sure that the required packages are installed:
+
+    ```sh
+    sudo dnf update --refresh
+    sudo dnf install -y git ninja-build cmake gcc-c++ SDL2-devel zlib-devel
+    ```
+
+    </details>
+
+2. **Windows only:** Open a “Developer Command Prompt for VS 2022”.
+
+
+3. Clone this repo:
+
+    ```sh
+    git clone https://github.com/DescentDevelopers/Descent3
+    ```
+
+4. Change directory into the clone:
+
+    ```sh
+    cd Descent3
+    ```
+
+5. **macOS only:** Install the rest of the prerequisites:
+
+    ```sh
+    brew bundle install
+    ```
+
+6. Run CMake:
+
+    ```sh
+    cmake --preset <platform>
```

Comment:
`<platform>` can directly be replaced by `[win32|mac|linux]` to avoid the extra line below

Reply:
I actually fixed this a while ago, but I forgot to mark this as resolved. Oops.